### DISCUSSION TITLE
Import interfaces not working after upgrade

### DISCUSSION THREAD

**Discussion by Stoutski** [netbox-community/netbox#17415] [General]
Hi,

 I get this error after updated to 4.1 when i try to import an interface:
<class 'TypeError'>

Field.__init__() got an unexpected keyword argument 'query_params'

Python version: 3.12.3
NetBox version: 4.1.0
Plugins: None installed

How can I fix this error?

**Comment by jeremystretch** [comment]
This will be addressed in v4.1.1; please see #17354.

**Reply by Stoutski** [reply]
Thanks I though it was for the devices import but there I don’t get this error only on the interfaces.
